_includes/2022_data/about.html

```diff
@@ -18,39 +18,39 @@ <h3 class="section-subheading text-muted"></h3>
                                 <div class="timeline-icon"><i class="fa fa-info"></i></div>
                                 <h3 class="title">Themes, Phases, Eligibility</h3>
                                 <p class="description">
-                                    Read more about our Themes, Phases, Eligibility
+                                    To know about our Themes, Phases, Eligibility
                                 </p>
-                                <a href="https://wiki.hyperledger.org/display/events/Hyperledger+Challenge+2022" class="page-scroll btn btn-sm">Learn More</a>
+                                <a href="https://wiki.hyperledger.org/display/events/Hyperledger+Challenge+2022" class="page-scroll btn btn-sm">Click here</a>
```

Comment:
[nitpick] `Click here` could be more intimidating than a polite `Learn more`. Maybe updating the wordings of this section is not needed at this stage of the event?

Reply:
Okay, I will revert back to the previous version